CRM/Upgrade/Incremental/sql/5.13.alpha1.mysql.tpl

```diff
@@ -1 +1,5 @@
 {* file to handle db changes in 5.13.alpha1 during upgrade *}
+
+-- dev/core#829 Swaziland has changed its name to Eswatini
+UPDATE civicrm_country SET name = 'Eswatini' WHERE name = 'Swaziland';
```

Comment:
Could you change this to WHERE id = 1203 since that is the primary key for the field.